deepmd/dpmodel/model/make_model.py

```
@@ -366,17 +367,18 @@ def _format_nlist(
             nnei: int,
             extra_nlist_sort: bool = False,
         ):
+            xp = array_api_compat.array_namespace(extended_coord, nlist)
             n_nf, n_nloc, n_nnei = nlist.shape
             extended_coord = extended_coord.reshape([n_nf, -1, 3])
             nall = extended_coord.shape[1]
```

Ruff

Local variable `nall` is assigned to but never used

Remove assignment to unused variable `nall`

(F841)